Header header
uint32[] hsvColorRanges # erster Range: [0-1], zweiter Range: [2-3] …
uint32[] quadCopterIds # erster Copter: [0], zweiter Copter: [1] …
---
uint8 error # 0 bei ok, != 0 sonst
